ingress/controllers/nginx/nginx/template.go

```diff
@@ -56,15 +56,9 @@ func (ngx *Manager) writeCfg(cfg nginxConfiguration, ingressCfg IngressConfig) (
 	conf["udpUpstreams"] = ingressCfg.UDPUpstreams
 	conf["defResolver"] = ngx.defResolver
 	conf["sslDHParam"] = ngx.sslDHParam
+	conf["customErrors"] = len(cfg.CustomHTTPErrors) > 0
 	conf["cfg"] = fixKeyNames(structs.Map(cfg))
 
-	buffer := new(bytes.Buffer)
-	err := ngx.template.Execute(buffer, conf)
-	if err != nil {
-		glog.Infof("NGINX error: %v", err)
-		return false, err
-	}
-
 	if glog.V(3) {
```

Comment:
suggest simply having a global --verbose=true and flipping log level to V(4) everywhere. That's been helpful for me at least, I almost never really want levelled logs. Just everything, or default. If you want to do that (not in this pr) you can copy how https://github.com/kubernetes/contrib/blob/master/ingress/controllers/gce/main.go#L111 is handled